### PR TITLE
done task16

### PR DESCRIPTION
**Pull request by macevilamv**

### CODE REVIEW

**Review by AntonNovosad — APPROVED** [review]

File: src/by/incubator/task16/Main.java [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    private static void displayTwoSkippedFirst(List<String> list) {
+        list.stream()
+             .skip(1)
+             .limit(2)
+             .forEach(System.out::println);
+    }
+
+    private static void displayElementsByTemplate(List<String> list) {
+        List<String> selected = list.stream()
+                                .filter(el -> el.equals("a1"))
```
Comment (AlexeyKatomachin) [inline src/by/incubator/task16/Main.java]: separate a1 variable into method variable

File: src/by/incubator/task16/Main.java [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+                                .collect(Collectors.toList());
+
+        System.out.println(selected);
+    }
+
+    private static boolean isAtLeastOneA1Present(List<String> list) {
+        return list
+                .stream()
+                .anyMatch("a1"::equals);
```
Comment (AlexeyKatomachin) [inline src/by/incubator/task16/Main.java]: separate a1 variable into method variable

File: src/by/incubator/task16/Main.java [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+    }
+
+    private static boolean isA7Present(List<String> list) {
+        return list
+                .stream()
+                .noneMatch("a7"::equals);
```
Comment (AlexeyKatomachin) [inline src/by/incubator/task16/Main.java]: separate a7 variable into method variable

File: src/by/incubator/task16/Main.java (hunk 1 above)
Comment (macevilamv) [reply src/by/incubator/task16/Main.java]: fixed

File: src/by/incubator/task16/Main.java (hunk 2 above)
Comment (macevilamv) [reply src/by/incubator/task16/Main.java]: fixed

File: src/by/incubator/task16/Main.java (hunk 3 above)
Comment (macevilamv) [reply src/by/incubator/task16/Main.java]: fixed

File: src/by/incubator/task2/JarTask.java [hunk 4]
```diff
@@ -1,23 +1,23 @@
...
-            Main instance = (Main) uploaded.newInstance();
-            instance.main(new String[]{});
-        } catch (InstantiationException e) {
-            throw new RuntimeException(e);
-        } catch (IllegalAccessException e) {
-            throw new RuntimeException(e);
-        }
-    }
-}
+//package by.incubator.task2;
```
Comment (AlexeyKatomachin) [inline src/by/incubator/task2/JarTask.java]: could you remove or uncommit ?

Comment (macevilamv) [reply src/by/incubator/task2/JarTask.java]: fixed

**Review by AlexeyKatomachin — APPROVED** [review]

**Pull request merged**